azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+# ASP.NET Core (.NET Framework)
+# Build and test ASP.NET Core projects targeting the full .NET Framework.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
+
+name: $(date:yyMMdd)$(rev:r)
+
+jobs:
+- job: CI
+  timeoutInMinutes: 360
+
+  pool:
+    vmImage: 'windows-latest'
```

Comment:
This mean it will only run on windows?

Reply:
Appears that the build template as of right now, only builds for Windows. I have a good setup for City Chain that I can migrate into this, see here: https://github.com/CityChainFoundation/city-chain/blob/master/azure-pipelines.yml

Reply:
ah I see will this also run tests?

Reply:
The "azure-pipelines" links to the shared build.yml which is same for all platforms, and the way I set it up for City Chain, is that on scheduled builds (supposed to be nightly), it will run all tests, but for normal builds it will only run test projects that begin with "City.*". Check line 47 in this file: https://github.com/CityChainFoundation/city-chain/blob/master/build.yml

Reply:
The script I have used for City Chain can be further improved, I know that there is a built-in task for publish to github now, while I'm using a third party extension.

Reply:
This is just specifying the vm image to use